app/views/offenders/show.html.haml

```diff
@@ -8,16 +8,19 @@
     %span.app-typography--body-1= @offender[:sid]
   .col.m5
     .app-typography--caption-above Offender Name
-    %span.app-typography--body-1= @offender[:name]
-.row.hide-on-med-and-down
-  .col.m5.offset-m1
-    .app-typography--caption-above Status
-    %span.app-typography--body-1= @offender[:status]
-  .col.m5
-    .app-typography--caption-above Offender Photo
     %span.app-typography--body-1
-      = link_to 'Show Photo', "http://docpub.state.or.us/OOS/imageLoader.jsp?idno=#{@offender[:sid]}", target: "_blank"
+      = @offender.include?(:name) ? @offender[:name] : "#{@offender[:first]} #{@offender[:last]}"
 .row.hide-on-med-and-down
+  - if @offender[:status]
+    .col.m5.offset-m1
+      .app-typography--caption-above Status
+      %span.app-typography--body-1= @offender[:status]
+  - if @offender[:jurisdiction] == :oregon
```

Comment:
Yeah, I agree that this should ideally fail in dev/test if the special values don't exist. Having an enum of expected values somewhere is probably the easiest way.